README.md

```diff
@@ -789,3 +790,49 @@ Response:
 must provide file_list field 
 ```
 
+# Autodetect App and File Type IDs
+
+## App type IDs
+
+These are the currently supported upload app type IDs:
+
+```
+fastq_reads_interleaved
+fastq_reads_noninterleaved
+sra_reads
+genbank_genome
+gff_genome
+gff_metagenome
+expression_matrix
+media
+fba_model
+assembly
+phenotype_set
+sample_set
+metabolic_annotation
+metabolic_annotation_bulk
```

Comment:
Hmm should we be supporting bulk upload here?

Reply:
Probably not, but it is worth considering that some bulk apps create a set vs the batch counterparts.. Like Genome vs GenomeSet...

Reply:
I'm more thinking it's just not going to work. If you pick N files and pick this app, you're just going to get N app instances in the narrative, when I assume what you want is a single app filled in with the N files.

Reply:
We should try it and see what happens / what the apps expects I guess. Like if takes a folder I'm not sure what'll happen...

Reply:
I don't think the staging area in the narrative supports it, so it's hidden for now anyway